modules/ztp-roll-out-the-configuration-changes.adoc

```diff
@@ -7,5 +7,5 @@
 = Roll out the configuration changes
 
 If any configuration changes were included in the upgrade due to implementing recommended changes, the upgrade process results in a set of policy CRs on the hub cluster in the `Non-Compliant` state. As of the {product-title} 4.10 release, these policies are set to `inform` mode and are not pushed to the spoke clusters without an additional step by the user. This ensures that potentially disruptive changes to the clusters can be managed in terms of when the changes are made, for example, during a maintenance window, and how many clusters are updated concurrently.
-
+// Please check line 9 for version. Does it need to be bumped to 4.11?
```

Comment:
Similar to the above note the context is slightly misleading. The version is static here as well, indicating when the change in policy mode was made. Maybe this for wording:
```
As of the 4.10 version of the `ztp-site-generate` container, these policies are ...
```